### ISSUE TITLE
Complete manual data entry in production ASpace

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/vt-arclight#339]

**Comment by marlo-longley** [comment]
Let's wait until we've resolved #326 to make sure we are set on the formatting.

**Comment by marlo-longley** [comment]
It's important that the HTML is copied directly from the staging ASpace interface because I made some fixes to the HTML there, which won't be reflected in the original source or file.

**Comment by marlo-longley** [comment]
This backup could easily become out of date if we continue to edit in ASpace. I think it's great to have a backup, but we should rely on pasting from the ASpace instance when we do the migration, as it may change in the course of QA.

**Comment by marlo-longley** [comment]
Blocked on #338

**Comment by laurensorensen** [comment]
When doing this, Column M in "USHMM collection name" should be added to Scope and content for film items (AKA this is a note to self): https://docs.google.com/spreadsheets/d/1QGcCtrGgASdvU6cHG7t09UVXMAk_DIN110zuWv42914/edit#gid=1206926021

**Comment by marlo-longley** [comment]
@laurensorensen re this note to self on Column M, if you only make this change in the production ASpace while you are migrating the content from staging, it will not get airtime on staging during QA / now, and it will be a difference between the data we are QA'ing and the production data.

**Comment by laurensorensen** [comment]
Thanks @marlo-longley I'll create a separate ticket. Commented on another related ticket which I will now delete, just seeing this!

**Comment by laurensorensen** [comment]
https://github.com/sul-dlss/vt-arclight/issues/368

**Comment by thatbudakguy** [comment]
@laurensorensen current data has been added to production ASpace, so everything there is ready for you to edit. going forward we're going to index EAD generated from production ASpace in both environments of the app, so #368 may not be necessary (or at least low-priority). lmk if that doesn't make sense

**Comment by thatbudakguy** [comment]
I can access the VT repository there using the `nta_app_prod` account — maybe it's a permissions issue? @marlo-longley do you know?

**Comment by laurensorensen** [comment]
This is all entered!
